daemon/algod/api/README.md

```diff
@@ -9,7 +9,35 @@ The API is defined using [OpenAPI v2](https://swagger.io/specification/v2/) in *
 
 1. Document your changes by editing **algod.oas2.json**
 2. Regenerate the endpoints by running **make generate**.
-3. Update the implementation in **server/v2/handlers.go**. It is sometimes useful to consult **generated/routes.go** to make sure the handler properly implements **ServerInterface**.
+3. Update the implementation in **server/v2/handlers.go**. It is sometimes useful to consult **generated/\*/\*/routes.go** to make sure the handler properly implements **ServerInterface**.
+
+### Adding a new V2 API
+When adding a new endpoint to the V2 APIs, you will need to add `tags` to the path. The tags are a way of separating our
+APIs into groups--the motivation of which is to more easily be able to conditionally enable and/or disable groups of
+endpoints based on the use case for the node.
+
+Each API in `algod.oas2.json`, except for some pre-existing `common` APIs, should have two tags.
+1. Either `public` or `private`. This controls the type of authentication used by the API--the `public` APIs use the
```

Comment:
If the goal is to have the data endpoints use admin authentication, but also be visible by the SDKs, something needs to change, since they will ignore `private` endpoints.

Reply:
I ended up switching the new data endpoints to public for this reason.

Reply:
Got it. Adding a link to #4900 in case anyone else is following this